quarkus/runtime/src/main/java/org/keycloak/quarkus/runtime/configuration/KeycloakConfigSourceProvider.java

```diff
@@ -63,6 +65,7 @@ public static void reload() {
 
     @Override
     public Iterable<ConfigSource> getConfigSources(ClassLoader forClassLoader) {
+        reload();
```

Comment:
Looks like we forgot to check if running in test mode?

Reply:
Done.